src/nwb_datajoint/common/common_filter.py

```diff
@@ -164,10 +164,10 @@ def filter_data_nwb(self, analysis_file_abs_path, eseries, filter_coeff, valid_t
         output_shape_list[electrode_axis] = len(electrode_ids)
         output_offsets = [0]
 
-        timestamp_size = sys.getsizeof(timestamps_on_disk[0])
-        timestamp_dtype = type(timestamps_on_disk[0])
-        data_size = sys.getsizeof(data_on_disk[0][0])
-        data_dtype = type(data_on_disk[0][0])
+        timestamp_size = timestamps_on_disk[0].itemsize
```

Comment:
Is `timestamps_on_disk` a numpy array? It seems to be a pointer to the h5 Dataset that contains timestamps. To use `itemsize` I think it has to be converted to a numpy array first.

Reply:
It is a pointer, but when one access an element, it becomes a numpy array, so this does work.

Reply:
Oh I see. Sorry, I thought the goal here was to get the size of the entire array - which is why I thought you had to first load the entire data to memory.